src/test/bin/quic_gtest.cpp

```diff
@@ -75,7 +75,10 @@ class QuicTestEnvironment : public ::testing::Environment {
             ASSERT_TRUE(QUIC_SUCCEEDED(MsQuic->GetInitStatus()));
             memcpy(&ServerSelfSignedCredConfig, SelfSignedCertParams, sizeof(QUIC_CREDENTIAL_CONFIG));
             memcpy(&ServerSelfSignedCredConfigClientAuth, SelfSignedCertParams, sizeof(QUIC_CREDENTIAL_CONFIG));
-            ServerSelfSignedCredConfigClientAuth.Flags |= QUIC_CREDENTIAL_FLAG_REQUIRE_CLIENT_AUTHENTICATION;
+            ServerSelfSignedCredConfigClientAuth.Flags |=
+                QUIC_CREDENTIAL_FLAG_REQUIRE_CLIENT_AUTHENTICATION |
+                QUIC_CREDENTIAL_FLAG_DEFER_CERTIFICATE_VALIDATION |
+                QUIC_CREDENTIAL_FLAG_INDICATE_CERTIFICATE_RECEIVED;
```

Comment:
We need to make sure we test (at least at the TLS unit test layer, if not here) both the defer and not modes.